src/Microsoft.AspNet.OData.Shared/Builder/ODataInstanceAnnotationContainer.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNet.OData.Builder
+{
+    /// <summary>
+    /// Default implementation for IODataInstanceAnnotationContainer
+    /// </summary>
+    public class ODataInstanceAnnotationContainer : IODataInstanceAnnotationContainer
+    {
+        private IDictionary<string, IDictionary<string, object>> instanceAnnotations;
+
+
+        /// <summary>
+        /// Default Constructor
```

Comment:
>Default Constructor [](start = 12, length = 19)

Initializes a new instance of the <see cref="ODataInstanceAnnotationContainer"/> class.